### PR TITLE
feat: regenerated sdks with new discovery credentials operations

### PR DESCRIPTION
**Pull request by dpopp07**
Regenerated the code for this weeks release. Added integration tests for the 5 new Discovery operations.

One of the names of the new discovery operations, `getCredentials()` conflicts with the `getCredentials()` operation in the base service. I renamed the discovery operation to `getSourceCredentials()`. If there are other suggestions for that name, I am happy to take them.

### CODE REVIEW

File: language-translator/v2-generated.ts [hunk 1]
```diff
@@ -21,9 +21,7 @@ import { getMissingParams } from '../lib/helper';
...
- * 
- * @deprecated Language Translator v3 is now available. The v2 Language Translator API will no longer be available after July 31, 2018. To take advantage of the latest service enhancements, migrate to the v3 API. View the [Migrating to Language Translator v3](https://console.bluemix.net/docs/services/language-translator/migrating.html) page for more information.
+ * ---  Language Translator v3 is [available](https://www.ibm.com/watson/developercloud/language-translator/api/v3/). See the [migration guide](https://console.bluemix.net/docs/services/language-translator/migrating.html).  ---  IBM Watson&trade; Language Translator translates text from one language to another. The service offers multiple domain-specific models that you can customize based on your unique terminology and language. Use Language Translator to take news from across the globe and present it in your language, communicate with your customers in their own language, and more.
```
Comment (mediumTaj) [inline language-translator/v2-generated.ts]: Will this line still show deprecation message for LTv2?

File: language-translator/v3.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ class LanguageTranslatorV3 extends BaseService {
    * @param {string} [options.password] - The password used to authenticate with the service. Username and password credentials are only required to run your application locally or outside of Bluemix. When running on Bluemix, the credentials will be automatically loaded from the `VCAP_SERVICES` environment variable.
    * @param {string} [options.iam_access_token] - An IAM access token fully managed by the application. Responsibility falls on the application to refresh the token, either before it expires or reactively upon receiving a 401 from the service, as any requests made with an expired token will fail.
    * @param {string} [options.iam_apikey] - An API key that can be used to request IAM tokens. If this API key is provided, the SDK will manage the token and handle the refreshing.
-   * @param {string} [options.iam_url] - An optional URL for the IAM service API. Defaults to 'https://iam.bluemix.net/identity/token'.
+   * @param {string} [options.iam_url] - An optional URL for the IAM service API. Defaults to 'https://iam.ng.bluemix.net/identity/token'.
```
Comment (mediumTaj) [inline language-translator/v3.ts]: This needs to be revised in the generator I think - the default URL is `https://iam.bluemix.net/identity/token`

File: language-translator/v2-generated.ts (hunk 1 above)
Comment (dpopp07) [reply language-translator/v2-generated.ts]: Probably not, I'll fix that and add it to the post-generation notes

File: language-translator/v3.ts (hunk 2 above)
Comment (dpopp07) [reply language-translator/v3.ts]: Revised, in all the SDKs. I will open a PR against the template soon

**Comment by dpopp07** [comment]
Comments addressed and unit tests added

**Pull request merged**